examples/abp_pcap_detection/abp_pcap_preprocessing.py

```diff
@@ -175,22 +175,24 @@ def round_time_kernel(timestamp, rollup_time, secs):
         count = data.shape[0]
 
         for col in req_cols:
-            x.set_meta(col, merged_df[col])
+            meta.set_data(col, merged_df[col])
 
         del merged_df
 
         seq_ids = cp.zeros((count, 3), dtype=cp.uint32)
-        seq_ids[:, 0] = cp.arange(x.mess_offset, x.mess_offset + count, dtype=cp.uint32)
+        seq_ids[:, 0] = cp.arange(0, count, dtype=cp.uint32)
         seq_ids[:, 2] = fea_len - 1
 
         # Create the inference memory. Keep in mind count here could be > than input count
-        memory = InferenceMemoryFIL(count=count, input__0=data, seq_ids=seq_ids)
+        memory = _messages.InferenceMemoryFIL(count=count, input__0=data, seq_ids=seq_ids)
```

Comment:
If this is a temporary change and the various subclasses of `TensorMemory` are going to be removed in a future PR, then feel free to ignore this comment.

I'm not sure if this is an improvement here, I think in general it is bad form to import the C++ impls directly, especially in an example where we are documenting how end-users should write custom stages.

I think we should either:
1. Continue using the Python implementations of the workflow-specific subclasses of `TensorMemory`
2. Stop using them, remove them and move everything to just use `TensorMemory` directly.